libkbfs/interfaces.go

```diff
@@ -1274,6 +1274,19 @@ const (
 	DiskBlockSyncCache
 )
 
+func (dbct DiskBlockCacheType) String() string {
+	switch dbct {
+	case DiskBlockSyncCache:
+		return "sync block cache"
+	case DiskBlockWorkingSetCache:
+		return "working set cache"
+	case DiskBlockAnyCache:
+		return "any disk block cache"
+	default:
+		return "unknown disk block cache"
+	}
+}
```

Comment:
Maybe just use the variable name in the returned string for easy grepping?

Reply:
👍